wcomponents-examples/src/main/resources/com/github/bordertech/wcomponents/examples/initImageEditExample.txt

```diff
@@ -0,0 +1,15 @@
+<script type="text/javascript">
+
+// Custom editor config turning on redact
+require(["wc/config"], function(config) {
+  config.set({redact:true, crop:false, format:'jpg'}, "wc/ui/imageEdit");
+});
+
+// When using a static WImage, you need to link the imageEditor and fileUploader
```

Comment:
As discussed this looks like something that needs to be tidied up, it should happen automagically.

Reply:
Agree. Have raised issue #1743

Reply:
Agree. Have raised issue #1743